src/main/java/de/tum/in/www1/artemis/web/rest/tutorialgroups/TutorialGroupResource.java

```diff
@@ -543,14 +543,16 @@ public enum TutorialGroupImportErrors {
      */
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
     public record TutorialGroupRegistrationImportDTO(@Nullable String title, @Nullable StudentDTO student, @Nullable Boolean importSuccessful,
```

Comment:
Is there a reason why you do not include the tutor login or the room?

Reply:
The Tutor Login was not included because I found it unintuitive to match tutors via CSV and would break the structure of the CSV format which is created by my other PR. I have adjusted the import process so that exporting tutor groups from the previous year can be directly used to import for the current year. While creating a tutor group, it makes more sense, in my opinion, to use tutors available in an Artemis course, which changes every year. Therefore, I designed it so that you can import all the fields and then select a tutor within Artemis. Initially, the room was planned to be imported and is included in the export. However, Artemis has a tutorial group design where a room can only be added if you create a tutorial group schedule, which is optional and requires fixed dates. This was causing additional problems, so addressing it should be part of a follow-up PR, as it would break the scope of this one.